prow/tide/tide_test.go

```diff
@@ -342,6 +342,14 @@ func TestAccumulate(t *testing.T) {
 			pendings:  []int{},
 			none:      []int{7},
 		},
+		{
+			pullRequests: map[int]string{7: "new", 8: "new"},
+			prowJobs:     []prowjob{},
+
+			successes: []int{8, 7},
+			pendings:  []int{},
+			none:      []int{},
+		},
 	}
```

Comment:
Please also add test cases for the changes you made. (A test case for `accumulateBatch` and for `takeAction`)

Reply:
test cases for accumulateBatch and takeAction added